sourcemash/items/models.py

```diff
@@ -7,6 +7,7 @@ class Item(db.Model):
     last_updated = db.Column(db.DateTime)
     author = db.Column(db.String(120))
     summary = db.Column(db.String(500))
+    text = db.Column(db.Text)
```

Comment:
Text doesn't take a min length; it will be too tough to pick a max length.

See: http://docs.sqlalchemy.org/en/rel_0_9/core/type_basics.html#sqlalchemy.types.Text